hostfactory/1.1/providerplugins/azurecc/scripts/src/cyclecloud_provider.py

```diff
@@ -461,12 +464,15 @@ def _get(name):
                 request_set["placementGroupId"] = template["attributes"].get("placementgroup")[1]
                                     
             add_nodes_response = self.cluster.add_nodes({'requestId': request_id,
-                                                         'sets': [request_set]})
+                                                        'sets': [request_set]})
             logger.info("Create nodes response: %s", add_nodes_response)
-            nodes_response = self.cluster.nodes_by_operation_id(operation_id=add_nodes_response["operationId"])
             
             with self.creation_json as requests_store:
-                requests_store[request_id]["allNodes"] = [x["NodeId"] for x in nodes_response["nodes"]]
+                 requests_store[request_id]["allNodes"] = [x.delayed_node_id.node_id for x in add_nodes_response.nodes]
```

Comment:
the "delayed_node_id" part may be better hidden as method on cluster.py  like "cluster.get_node_id(x)"